domador.js

```diff
@@ -659,4 +659,13 @@ Domador.prototype.isVisible = function isVisible (el) {
   return visible;
 };
 
+Domador.prototype.openCodeFence = function openCodeFence (el) {
+  var fencinglanguage = this.options.fencinglanguage(el);
+  var child = el.childNodes[0];
+  if (child && child.tagName === 'CODE') {
+    fencinglanguage = fencinglanguage || this.options.fencinglanguage(el.childNodes[0]);
```

Comment:
hmm, internal "code" fencinglanguage should override parent element fencinglanguage, right?

Sorry, I'm not sure I follow... can't think of a way to move the || check into line 665's conditional.